starters/graphql-serverless-contentful/src/models/PostModel.ts

```diff
@@ -0,0 +1,65 @@
+import { client } from '../utils/contentful';
```

Comment:
I see that you added a PostModel.test file but it currently empty. 
I think we should either add a test or remove it completely.